[Main Thread/main.py] Error: Traceback (most recent call last):
  File "C:\GitHub\vis\gd\main.py", line 260, in <module>
    main()
  File "C:\GitHub\vis\gd\main.py", line 65, in main
    call_handle_page_function(val)
  File "C:\GitHub\vis\gd\main.py", line 101, in call_handle_page_function
    handle_function(val)
  File "C:\GitHub\vis\gd\main.py", line 216, in handle_created_levels_page
    level_name=level_info[level_info.index('levels\\')+1:]
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: substring not found
